torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3979,10 +4004,10 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.linalg.det,
            aliases=('det', ),
            dtypes=floating_and_complex_types(),
+           backward_dtypes=floating_types(),
```

Comment:
Add a comment (ideally with a link to an issue) above these "backward_dtypes" lines where supports_complex_autograd was removed, just to keep the lack of complex autograd support visible